### ISSUE TITLE
NoMethodError in Shoppe::Settings#edit

### ISSUE THREAD

**Issue by raybesiga** [tryshoppe/shoppe#214]
When I try to access the settings in the admin dashboard, I get the following exception:

Showing /Users/raybesiga/.rvm/gems/ruby-2.2.2/gems/shoppe-1.0.7/app/views/shoppe/settings/edit.html.haml where line #12 raised:

```ruby
undefined method `settings_label' for #<#<Class:0x007ff0bb83ac30>:0x007ff0bcc1b7e0>

      - fields.each do |f|
        %dl.sided
          %dt= settings_label f
          %dd
            = settings_field f
            - if v = t("shoppe.settings.help")[f]
```

Did a gem update break something?

**Comment by raybesiga** [comment]
Have you come across this error @deanperry ?

**Comment by deanperry** [comment]
I've not seen that error, no but I'll take a look and see if I can replicate it

**Comment by raybesiga** [comment]
It happens when you follow the tutorial but use the latest version of the
Shoppe gem. Also noticed that having @product_category.includes from the
example shoppe throws exceptions. Any chance the example shop will be
updated to use the current Gem?

On Monday, June 22, 2015, Dean Perry <notifications@github.com> wrote:

> I've not seen that error, no but I'll take a look and see if I can
> replicate it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tryshoppe/shoppe/issues/214#issuecomment-114212024>.
>


-- 
With optimism,

Raymond Besiga
h: http://raybesiga.com
w: http://sparkpl.ug  <http://sparkpl.ug/>
m: +256-791-160903

Stalk me: http://twitter.com/raybesiga

**Comment by deanperry** [comment]
I've just tried this on a new application and I'm not getting the error. I've updated the example shop for Rails 4.2.3 and the latest version of Shoppe - https://github.com/tryshoppe/shoppe-example

**Comment by raybesiga** [comment]
Thanks so much!

On Sat, Jun 27, 2015 at 11:29 PM, Dean Perry <notifications@github.com>
wrote:

> Closed #214 <https://github.com/tryshoppe/shoppe/issues/214>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tryshoppe/shoppe/issues/214#event-342068623>.
>



-- 
With optimism,

Raymond Besiga
h: http://raybesiga.com
w: http://sparkpl.ug  <http://sparkpl.ug/>
m: +256-791-160903

Stalk me: http://twitter.com/raybesiga
